src/test/scala/nl/knaw/dans/easy/fedora2vault/fixture/TestSupportFixture.scala

```diff
@@ -15,11 +15,16 @@
  */
 package nl.knaw.dans.easy.fedora2vault.fixture
 
+import better.files.File
 import org.scalatest._
 
-trait TestSupportFixture extends FlatSpec
+trait TestSupportFixture extends FlatSpec with TimeZoneFixture
```

Comment:
Probably this project is not yet upgraded to the new parents. Please do so and use `AnyFlatSpec` instead as `FlatSpec` is now deprecated.